config/manifests/bases/nfd-operator.clusterserviceversion.yaml

```diff
@@ -13,7 +13,7 @@ metadata:
           },
           "spec": {
             "operand": {
-              "image": "gcr.io/k8s-staging-nfd/node-feature-discovery:master",
+              "image": "k8s.gcr.io/nfd/node-feature-discovery:v0.11.0-minimal",
```

Comment:
Think we should use master-minimal here.

Reply:
ok